DataProducts/inc/STMTestBeamHeaders.hh

```diff
@@ -0,0 +1,90 @@
+#ifndef DataProducts_STMTestBeamHeaders_hh
```

Comment:
Is this class intended as a prototype for eventual use with full Mu2e readout of STM data?  Or is the testbeam data so specific it won't get reused?

Reply:
I didn't intend for this to become used in the full Mu2e readout. The same is true for the STMTestBeamEventInfo class. The final data packets will look pretty similar to this but I think that will all live in the online world

Reply:
Thanks.  I will follow up on this outside of this ticket.